driver-core/src/test/java/com/datastax/driver/core/utils/CassandraVersion.java

```diff
@@ -34,7 +34,12 @@
     /**
      * @return The minor version required to execute this test, i.e. "0"
      */
-    int minor() default 0;
+    int minor() default -1;
+
+    /**
+     * @return Whether or not the exact major version is exactly required to run this test.
+     */
+    boolean[] exactRequired() default {false};
```

Comment:
Why do you need this attribute to be a boolean array? I t seems a simple boolean is enough.